web/templates/registration/login.html

```diff
@@ -1,50 +1,31 @@
 {% extends "web/base.html" %}
-{% load bootstrap3 %}
+{% load django_bootstrap5 %}
 {% load i18n %}
-{% block flash %}
-    {% bootstrap_messages %}
-    {% if form.errors %}
-        <div class="alert alert-danger" role="alert">
-            <p>
-                {% trans "Your login credentials didn't work. Please try again." %}
-            </p>
-        </div>
-    {% endif %}
-{% endblock flash %}
```

Comment:
One question: it looks like this got removed (rather than moved somewhere else) right? Do we not need it anymore?

Reply:
The form contains the error, which I was satisfied with.  It's less that the error message was removed, but that It isn't reported in a alert messages block.